operations/iscsi_sbps/Adjusting_Selective_iSCSI_Node_Personalization.md

```diff
@@ -0,0 +1,62 @@
+# Adjusting Selective iSCSI Node Personalization
+
+## Overview
+
+In CSM 1.6.0, all worker nodes are configured and enabled as iSCSI SBPS targets during [worker node personalization](https://github.com/Cray-HPE/docs-csm/blob/release/1.7/operations/configuration_management/iSCSI_SBPS_Node_Personalization.md)
```

Comment:
When linking to other pages in `docs-csm` inside the same branch, links should never be full `https` links. They should be relative path links to the markdown file. All links in this page appear to have this problem.